website/docs/secrets/setup-secrets-operator.mdx

```diff
@@ -3,12 +3,125 @@ title: Setup Secrets Operator
+        - The Job CR, follow [this guide](/cluster-management/getting-started/#add-a-cluster-bootstrap-config) for more information on ClusterBootstrapConfig
+
+            <details><summary>Expand to view </summary>
+                <CodeBlock
+                title="clusters/management/capi/boostrap/sops-bootstrap-job.yaml"
+                className="language-yaml"
+                >
+                    {SopsBootstrapJob}
+                </CodeBlock>
+            </details>
+
+        - Template Updates
+
+            In order to install sops we need some modifications to the template to allow it to create a [Kustomization](https://fluxcd.io/flux/guides/mozilla-sops/#configure-in-cluster-secrets-decryption) for reconciling the secrets on the cluster for sops during cluster creation for the secrets path and run the `ClusterBootstrapConfig` job
+
+            It will refer to `sops` location in the root of cluster files
+
+            - Template Updates
+                - It should have annotation `templates.weave.works/sops-enabled: "true"` to enable WGE to create the kustomization in cluster creation
+                - It should have the following values that’s needed for **kustomization**
+
+                - GitOps template with sops enabled (check how to add the annotations below)
+
+                    <details><summary>Expand to view </summary>
+                        <CodeBlock
+                        title="clusters/management/capi/boostrap/template.yaml"
+                        className="language-yaml"
+                        >
+                        {TemplateParams}
+                        </CodeBlock>
+                    </details>
+
+                - It should have the following annotations under `GitOpsCluster` to be used in the bootstrap job
+
+                    <details><summary>Expand to view </summary>
+                        <CodeBlock
+                        title="clusters/management/capi/boostrap/template.yaml"
+                        className="language-yaml"
+                        >
+                        {TemplateAnnotations}
+                        </CodeBlock>
+                    </details>
+
+- **Installation Steps**
+    - Go to `Clusters` from the side bar and `Create A Cluster`
+    - Select your Sops template (must have the annotation and parameters)
+        ![ESO Cluster](./imgs/sops1.jpg)
+    - Fill the data, Create and Merge the PR
+    - You should expect the following after the cluster is ready
+      - A leaf cluster created with flux & sops bootstrapped
+      - A secret created on leaf cluster `sops-gpg` to decrypt secrets
+      - A secret created on leaf cluster `sops-gpg-pub` to encrypt secrets
+      - A Kustomization with `decryption` defined in it to `sops` location in the cluster repo location
+      - A PR is created to the cluster repo with the public key and sops creation rules (optional)
+      - Added Role for the public key to be accessed through management cluster
```

Comment:
After the installation steps you can point the user to the UI pages so they can start using the create form